### ISSUE TITLE
Should not present "no imagery at this zoom" tiles

### ISSUE THREAD

**Issue by bgirardot** [mapswipe/Mapswipe-app#8]
My small block of offline work included 30 screens of the camera with a line through it graphic indicating there is no imagery available.

However, 30 screens = 180 tiles = 540 taps to indicate bad imagery. I couldn't do that so I just swiped past them.

I think the app or back end should be able to determine if there is actually no imagery (as opposed to bad imagery) so we do not ask people tap so many times for something that we should be able to figure out programmatically.

**Comment by pimdeleted** [comment]
Agreed. We can compare the no imagery picture bytes since the http request
returns 200
On Wed, Jul 13, 2016 at 7:10 PM Blake Girardot <notifications@github.com>
wrote:

> My small block of offline work included 30 screens of the camera with a
> line through it graphic indicating there is no imagery available.
>
> However, 30 screens = 180 tiles = 540 taps to indicate bad imagery. I
> couldn't do that so I just swiped past them.
>
> I think the app or back end should be able to determine if there is
> actually no imagery (as opposed to bad imagery) so we do not ask people tap
> so many times for something that we should be able to figure out
> programmatically.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mapswipe/Mapswipe-app/issues/8>, or mute the thread
> <https://github.com/notifications/unsubscribe/AE_WF3Cvz0Ubcg7u-DNBERmERQo1jowuks5qVZqLgaJpZM4JMBOJ>
> .
>
-- 
Pim de Witte
(+1) 786 201-9489
pimdewitte95@gmail.com
